lib/brakeman/checks/check_forgery_setting.rb

```diff
@@ -7,67 +7,69 @@
 class Brakeman::CheckForgerySetting < Brakeman::BaseCheck
   Brakeman::Checks.add self
 
-  @description = "Verifies that protect_from_forgery is enabled in ApplicationController"
+  @description = "Verifies that protect_from_forgery is enabled in direct subclasses of ActionController::Base"
 
   def run_check
-    app_controller = tracker.controllers[:ApplicationController]
-    return unless app_controller and app_controller.ancestor? :"ActionController::Base"
+    tracker.controllers
+        .select { |_, controller| controller.parent == :"ActionController::Base" }
+        .each do |name, controller|
 
-    if tracker.config.allow_forgery_protection?
-      warn :controller => :ApplicationController,
-        :warning_type => "Cross-Site Request Forgery",
-        :warning_code => :csrf_protection_disabled,
-        :message => "Forgery protection is disabled",
-        :confidence => CONFIDENCE[:high],
-        :file => app_controller.file
+      if tracker.config.allow_forgery_protection?
+        warn :controller => name,
```

Comment:
It will only generate a warning for a controller that directly inherits from `ActionController::Base`. Will that be an issue? I would imagine that there may be apps that has more than one controllers that inherits from `ActionController::Base`. Like say the `ApplicationController` and `AdminController`.